source/fe/fe_values.cc

```diff
@@ -3493,6 +3493,10 @@ maybe_invalidate_previous_present_cell (const typename Triangulation<dim,spacedi
       // at least subscribe to the triangulation to get notified of
       // changes
       tria_listener =
+        cell->get_triangulation().signals.mesh_movement.connect
+        (std_cxx11::bind (&FEValuesBase<dim,spacedim>::invalidate_present_cell,
+                          std_cxx11::ref(static_cast<FEValuesBase<dim,spacedim>&>(*this))));
+      tria_listener =
```

Comment:
@bangerth It turns out that I introduced a bug here since I tried to put the tria_listener to two different signals: On my system I see the following tests failing:
```
        2566 - fe/nedelec.debug (Failed)
        2567 - fe/nedelec.release (Failed)
        2619 - fe/bdm_2.debug (Failed)
        2620 - fe/bdm_2.release (Failed)
        2703 - fe/rt_2.debug (Failed)
        2704 - fe/rt_2.release (Failed)
```
The problem is that the triangulation tries to notify a FEValues object through invalidate_present_cell but the FEValues object was already deleted. You know the `tria_listener` concept better than me: Do we need to add a second variable like `tria_listener` (I would make it a vector) to be able to control both listeners? I thought that the listener would add the two signals by `operator=` but apparently I was wrong.